docs/labs/10-Timer.md

```diff
@@ -609,10 +609,35 @@ For example,
 you could add a label that is continually updated or set in `actionPerformed`
 that would update the number of green squares that you have removed from the screen
 (see the version on the left).
+
+```java
```

Comment:
please provide some intro to this code snippet.

Reply:
I added some intro to the code in the latest pull request